client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -161,6 +169,16 @@ const FileInfoCard: FunctionComponent< FileInfoCardProps > = ( {
 		prepareDownload( siteId, item.period, fileInfo.manifestFilter, fileInfo.dataType );
 	}, [ dispatch, fileInfo, item.period, prepareDownload, siteId ] );
 
+	const restoreFile = useCallback( () => {
+		// Reset checklist
+		dispatch( setNodeCheckState( siteId, '/', 'unchecked' ) );
```

Comment:
It could be better to have an action like `JETPACK_BACKUP_BROWSER_CHECK_STATE_RESET` to reset the rewind.browser to its initial state. It can be done in a separate PR.